packages/base-plugin/lib/plugin.js

```diff
@@ -43,11 +49,16 @@ class BasePlugin {
   /**
    * @param {string} name
    * @param {Record<string,unknown>} [cliArgs]
+   * @param {string|null} driverId
    */
-  constructor(name, cliArgs = {}) {
+  constructor(name, cliArgs = {}, driverId = null) {
+    super();
+    if (driverId) {
+      this.updateLogPrefix(`${generateDriverLogPrefix(this)} (for driver ${driverId})`);
```

Comment:
I would prefer shorter log prefixes, otherwise this `(for driver ` part would be repeated for each log line thus making less space for useful message payload. What if we format it as `${generateDriverLogPrefix(this)}<${driverId}>`?